### PR TITLE
Add page for installation instructions for Linux command-line

### PR DESCRIPTION
**Pull request by waltlillyman**
Adds new page, `manual_cli.md`, and link to it from existing page, `documentation/installation/manual.md`

### CODE REVIEW

File: documentation/installation/manual_cli.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+...
+  inflating: config_flow.py          
+  inflating: ws_api_handlers.py      
+   creating: .translations/
+  inflating: .translations/nl.json   
+  inflating: .translations/fr.json   
+...
+  inflating: hacsbase/task_factory.py  
+  inflating: hacsbase/__init__.py    
+  inflating: hacsbase/exceptions.py
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md]: Outputs should not be part of a guide

File: documentation/installation/manual_cli.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+### Step 2 - Download the latest HACS release
+
+At your Linux command-line, type `wget ` followed by a space, then paste the URL from your clipboard. Press Enter to download `hacs.zip` to your current directory. (`xx` and `y`, below, will be the actual latest-version numbers in the URL):
+```
+user@host:~$ wget https://github.com/hacs/integration/releases/download/0.xx.y/hacs.zip
+--2020-01-01 12:58:13--  https://github.com/hacs/integration/releases/download/0.xx.y/hacs.zip
+...
+Connecting to github.com (github.com) ... connected.
+...
+HTTP request sent, awaiting response... 200 OK
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md]: Outputs should not be part of a guide

File: documentation/installation/manual_cli.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+![install2](/img/install2.png)
+
+Under Assets, right-click `hacs.zip` and copy that URL to your clipboard.
+
+### Step 2 - Download the latest HACS release
+
+At your Linux command-line, type `wget ` followed by a space, then paste the URL from your clipboard. Press Enter to download `hacs.zip` to your current directory. (`xx` and `y`, below, will be the actual latest-version numbers in the URL):
+```
+user@host:~$ wget https://github.com/hacs/integration/releases/download/0.xx.y/hacs.zip
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md]: prefix in for the commands should not be a part of it like `user@host:~$`

File: documentation/installation/manual_cli.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+id: manual_cli
+title: Installation, Linux command-line
+description: "Manual installation steps using the Linux command-line"
+---
+
+This guide will help you install HACS using the Linux command-line on your Home Assistant server.
+
+### Step 1 - Get the URL for the latest HACS release
+
+In your browser, visit the [HACS latest release page](https://github.com/custom-components/hacs/releases/latest).
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md]: For a fully CLI installation, you should not need to go there.
There should be a way to get that URL by using CLI only.

File: documentation/installation/manual_cli.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+For this example, `<config-dir>` will be `/path/to/homeassistant/config`
+
+Change-directory to your Home Assistant configuration directory:
+```
+user@host:~$ cd /path/to/homeassistant/config
+user@host:/path/to/homeassistant/config$ 
+```
+
+Create subdirectories, `custom_components/hacs`. If the subdirectories don't exist, this command will create them:
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md]: This is not needed, you can just use the `-d` flag when unzipping.
Like `unzip hacs.zip -d hacs`

**Comment by ludeeus** [comment]
I made a install script, can you rework this section to use that?

```bash
curl -sSL https://hacs.xyz/install | bash -
```

**Comment by ludeeus** [comment]
@waltlillyman Are you continuing on this?

**Comment by waltlillyman** [comment]
Updating this now.

File: documentation/installation/manual_cli.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+On Hassio/Docker the final location will be `/config/custom_components/hacs`
+
+On Hassbian(venv) the final location will be `/home/homeassistant/.homeassistant/custom_components/hacs`
+
+For this example, `<config-dir>` will be `/path/to/homeassistant/config`
+
+Change-directory to your Home Assistant configuration directory:
+```
+cd /path/to/homeassistant/config
```
Comment (ludeeus) [inline documentation/installation/manual_cli.md:23]: This part is not needed, the script handles it.

If no dir is found, the script will tell the user to do this as well.

**Pull request merged**